src/Keboola/Console/Command/MassProjectQueueMigration.php

```diff
@@ -0,0 +1,187 @@
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class MassProjectQueueMigration extends Command
+{
+    const ARGUMENT_MANAGE_TOKEN = 'manage-token';
+    const ARGUMENT_CONNECTION_URL = 'connection-url';
+    const ARGUMENT_SOURCE_FILE = 'source-file';
+
+    const FEATURE_QUEUE_V2 = 'queuev2';
+    const COMPONENT_QUEUE_MIGRATION_TOOL = 'keboola.queue-migration-tool';
+    const JOB_STATES_FINAL = ['success', 'error', 'terminated', 'cancelled'];
+
+    protected function configure()
+    {
+        $this
+            ->setName('manage:mass-project-queue-migration')
+            ->setDescription('Mass project migration to Queue v2')
+            ->addArgument(self::ARGUMENT_MANAGE_TOKEN, InputArgument::REQUIRED, 'Manage token')
+            ->addArgument(self::ARGUMENT_CONNECTION_URL, InputArgument::REQUIRED, 'Connection url')
+            ->addArgument(self::ARGUMENT_SOURCE_FILE, InputArgument::REQUIRED, 'Source file with project ids')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $manageToken = $input->getArgument(self::ARGUMENT_MANAGE_TOKEN);
+        $kbcUrl = $input->getArgument(self::ARGUMENT_CONNECTION_URL);
+        $sourceFile = $input->getArgument(self::ARGUMENT_SOURCE_FILE);
+        $output->writeln(sprintf('Fetching projects from "%s"', $sourceFile));
+
+        $manageClient = new Client([
+            'token' => $manageToken,
+            'url' => $kbcUrl,
+        ]);
+
+        $logger = new ConsoleLogger($output);
+        $queueApiUrl = str_replace('connection', 'queue', $kbcUrl);
+
+        $projects = $this->parseProjectIds($sourceFile);
+        $output->writeln(sprintf('Migrating "%s" projects', count($projects)));
+
+        $migrationJobs = [];
+        foreach ($projects as $projectId) {
+            // set queuev2 project feature
+            $projectRes = $manageClient->getProject($projectId);
+            if (!in_array(self::FEATURE_QUEUE_V2, $projectRes['features'])) {
+                $manageClient->addProjectFeature($projectId, self::FEATURE_QUEUE_V2);
+            }
+
+            $storageToken = $this->createStorageToken($manageClient, $projectId);
+            $storageClient = new StorageClient([
+                'token' => $storageToken,
+                'url' => $kbcUrl,
+            ]);
+            $componentClient = new Components($storageClient);
+            $configuration = new Configuration();
+            $configuration
+                ->setChangeDescription('Created configuration')
+                ->setComponentId(self::COMPONENT_QUEUE_MIGRATION_TOOL)
+                ->setName('Queue migration');
+
+            $configuration = $componentClient->addConfiguration($configuration);
```

Comment:
ta konfigurace tam na něco je?

Reply:
Jo, vidis to som zabudol zmienit, ze ten public client ma povinne `configId` pri vytvarani jobu, ale skusim ho dat ako null a poslat tam `configData` este.

Na druhej strane som rozmyslal, ci tam tu konfiguraciu nenachat, ked job skonci errorom (pre dalsie rucne spustenia), ale kedze odstranujem pri errore aj tu feature queuev2... Ne, je to blbost :D 
Skusim to cez tie configData este.

Reply:
No to je blbost - configId může být prázdný https://github.com/keboola/job-queue-api-php-client/blob/d3e6b82cd0299e73d9085e5ef32ce3660366848b/tests/ClientFunctionalTest.php#L29 akorát by to mělo být teda nullable, aby to bylo "čistý"

Reply:
JJ, funguje to s prazdnym stringom "nastastie"

Reply:
Asi to rovno fixnem aj teda.